src/main/java/com/appoptics/integrations/kafka/broker/KafkaAppopticsReporter.java

```diff
@@ -25,10 +28,33 @@ public synchronized void init(VerifiableProperties props) {
         }
 
         String token = props.getString("appoptics.token");
-        String source = props.getString("appoptics.agent.identifier");
+        List<Tag> tags = new ArrayList<Tag>();
+
+        String source = props.getString("appoptics.agent.identifier", "");
+        if (!source.equals("")) {
+            tags.add(new Tag("source", source));
+        }
+
+        String customTags = props.getString("appoptics.tags", "");
```

Comment:
Would it make sense to make keys like `appoptics.*` static final constants? or even better an enum with string values?